parquet/write_read_test.go

```diff
@@ -12,9 +12,16 @@ import (
 	"github.com/cloudquery/plugin-sdk/v3/schema"
 )
 
+var pqTestOpts = schema.TestSourceOptions{
+	// persisted as timestamp[ms]:
+	SkipTimestamps: true,
```

Comment:
These types are all persisted as timestamp[ms] so we choose to skip the test/compare logic for them.

Reply:
hmm, can't we handle that? it's pretty important to check that we handle all timestamps correctly 🤔 also, are they persisted as timestamp[ms] or timestamp[us]?

Reply:
All timestamps are "persisted" as `timestamp[ms, tz=UTC]` in pqarrow for some reason. Maybe it's parquet's native timestamp format I need to check.

Reply:
@hermanschaaf  There's a timestamp coercion feature in pqarrow which defaults to false but they still seem to get persisted only as ms.

There's also a special case which always coerces second percision to ms:
```go
	// the user implicitly wants timestamp data to retain it's original time units,
	// however the arrow seconds time unit cannot be represented in parquet, so must
	// be coerced to milliseconds
	if typ.Unit == arrow.Second {
		logicalType = arrowTimestampToLogical(typ, arrow.Millisecond)
	}
```\